portal-api/src/crypto/utils.js

```diff
@@ -61,7 +61,6 @@ function issueJWTWithExpiryAndPayload({ user, sessionIdentifier = crypto.randomB
 
   const payload = {
     sub: _id,
-    iat: Date.now(),
```

Comment:
Just to double-check I've understood:
- `jsonwebtoken` was calculating the `exp` claim from the `iat` and `expiresIn` that we specified (and the `iat` was out by a factor of 1000 so our `exp` claim was incorrect)
- by not providing the `iat` ourselves:
  - `jsonwebtoken` will calculate `exp` from `expiresIn` correctly
  - `jsonwebtoken` will still calculate the `iat` claim for the JWT itself and include it

Is that right?

Reply:
Exactly 
https://github.com/auth0/node-jsonwebtoken

Generated jwts will include an `iat` (issued at) claim by default unless `noTimestamp` is specified. If `iat` is inserted in the payload, it will be used instead of the real timestamp for calculating other things like `exp` given a timespan in `options.expiresIn`.